api/.pylintrc

```diff
@@ -548,7 +548,9 @@ valid-metaclass-classmethod-first-arg=cls
 [DESIGN]
 
 # Maximum number of arguments for function / method.
-max-args=6
+# 2020/07/07:SS We very often have a lot of arguments, and that's just how it is. We're spending too much
+# time adding lint ignores.
+max-args=10
```

Comment:
I'm not opposed to this but typically with long parameter lists there's groupings of parameters that make sense: https://refactoring.com/catalog/introduceParameterObject.html
(If there's consensus on this, let's not address it in this PR, wait for refactor task)

Reply:
I've added this comment to the tech debt ticket.